docs/display/difference.md

```diff
@@ -6,20 +6,21 @@ title: Difference
 This indicates the difference between two date-time in the specified unit.
 
 To get the difference in milliseconds, use `dayjs#diff`.
+
 ```js
-const date1 = dayjs('2019-01-25')
-const date2 = dayjs('2018-06-05')
-date1.diff(date2) // 20214000000 default milliseconds
+const date1 = dayjs('2019-01-25');
+const date2 = dayjs('2018-06-05');
+date1.diff(date2); // 20214000000 default milliseconds
 ```
 
 To get the difference in another unit of measurement, pass that measurement as the second argument.
 
 ```js
-const date1 = dayjs('2019-01-25')
-date1.diff('2018-06-05', 'month') // 7
+const date1 = dayjs('2019-01-25');
+date1.diff("2018-06-05", 'month'); // 7
```

Comment:
also the quotes here, and remove the semicolon as well pls

Reply:
Updated files, I left some semicolons in other files too! Sorry, my global prettier automatic formatted the code and I did not check it up your style. Thanks for the trouble!